src/json/json_value.cc

```diff
@@ -450,6 +450,19 @@ auto JSON::operator-=(const JSON &substractive) -> JSON & {
   }
 }
 
+[[nodiscard]] auto JSON::try_at(const JSON::String &key) const
+    -> std::optional<JSON> {
+  assert(this->is_object());
+
+  const auto &object{std::get<Object>(this->data)};
+  const auto value{object.data.find(key)};
+
+  if (value == object.data.end()) {
```

Comment:
```suggestion
  if (value == object.data.cend()) {
```

Just to make the intention clearer?